.github/workflows/scheduled-build.yml

```diff
@@ -0,0 +1,18 @@
+name: Build images for Docker Hub
+on:
+  schedule:
+    - cron: '0 18 * * 1' # every monday at 18:00
+
+jobs:
+  build:
+    runs-on: ubuntu-18.04
+    steps:
+      - uses: actions/checkout@v1
+      - name: Docker Login
+        run: docker login -u "${{ secrets.DOCKER_USER }}" -p "${{ secrets.DOCKER_PASSWORD }}"
+      - name: Set Tag
+        run: echo ::set-env name=DATE::$(date "+%Y%m%d")
+      - name: Build Image
+        run: docker build --build-arg ubuntu_version=18.04 -t projectsyn/suc-ubuntu-bionic:$DATE .
```

Comment:
A build for 20.04 would also be great.

Reply:
Good idea. You can create a new job under `jobs:` so they will run in parallel.

Reply:
Created another job and tested the workflow again (check GitHub Actions or Docker Hub).